apoiase/app.py

```diff
@@ -20,17 +20,19 @@ def home():
     if response.status_code == 200:
         page = json.loads(response.text)
         users = page.get('supports', {}).get('users', [])
-        names = map(serializer, users)
-        names = [name for name in names if name]
+        users = filter(filter_private_supporters, users)
+        names = list(map(serializer, users))
         return jsonify(names)
 
 
-def serializer(users):
-    private=users.get('privateSupport')
-    if not private:
-        return dict(name=users.get('_id').get('name'))
+def filter_private_supporters(apoiase_users):
```

Comment:
so yeah! we can use the lambda approach, I'm not a fan of lambda functions so that's why I defined the function used in the filter bit...

>Given that user already has a boolean for that, we can even go for filter(lambda u: u.get('privateSupport'), users)

this gives the same result as the one I did above... I can change it =)
@anaschwendler what do you think? go for lambda?